### PR TITLE
Document UASTC_RG channel id and rewrite section on UASTC_ETC1S DFDs.

### PR DESCRIPTION
**Pull request by MarkCallow**

### CODE REVIEW

File: ktxspec.adoc [hunk 1]
```diff
@@ -970,29 +981,41 @@ The ETC1S image format is indicated by `colorModel` `KDF_DF_MODEL_ETC1S`
...
+as detailed in the following table of valid channel id combinations.
 +
-The only valid combinations of more than one slice are `RGB` + `AAA`
-or `RRR` + `GGG` with `RGB` or `RRR` being the first slice.
+[width=100%,align=center,cols="<65,<35",options=header]
+|===
+| Combination | Description
+|`KHR_DF_CHANNEL_ETC1S_RGB` | One slice, opaque color.
+|`KHR_DF_CHANNEL_ETC1S_RGB` + `KHR_DF_CHANNEL_ETC1S_AAA` | Two slices,
+                                                           color + transparency.
```
Comment (lexaknyazev) [inline ktxspec.adoc:1011]: For consistency, consider using "alpha" instead of "transparency" here.

Comment (MarkCallow) [reply ktxspec.adoc]: Thanks. Done.

File: ktxspec.adoc [hunk 2]
```diff
@@ -970,29 +981,41 @@ The ETC1S image format is indicated by `colorModel` `KDF_DF_MODEL_ETC1S`
...
+| Id | Value | Type 
+|`KHR_DF_CHANNEL_ETC1S_RGB` | 0  | 3 component: opaque color. RGB
+                                   components in the rgb channels.
+|`KHR_DF_CHANNEL_ETC1S_RRR` | 3  | 1 component: R component replicated
+                                   in all 3 rgb channels for better
+                                   compression results.
+|`KHR_DF_CHANNEL_ETC1S_GGG` | 4  | Not used independently. 1 component:
+                                   G component replicated in all 3 rgb
+                                   channels
+|`KHR_DF_CHANNEL_ETC1S_AAA` | 15 | Not used independent;y. 1 component:
```
Comment (abwood) [inline ktxspec.adoc:998]: Minor nit; typo in independently

Comment (MarkCallow) [reply ktxspec.adoc]: Fixed. Thanks.

**Pull request merged**